### PR TITLE
Doc/improve readability

### PR DESCRIPTION
**Pull request by cvmiert**
Reviewers: @fabiotc @rdohms @WyriHaximus @renatomefi @frankkoornstra @agustingomes @cvmiert

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Documentation?    | yes
| Dockerfile change?| no
| Build feature?    | no
| Apply CVE Patch?  | no
| Remove CVE Patch? | no

## Changelog

- Update reference to current alpine versions
- Improve README.md readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ All images are based on their official variants, being:
 
 ## Alpine Linux situation
 
-Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose between its versions, at this moment being `3.7` or `3.8`.
+Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose between its versions, at this moment being `3.8` or `3.9`.
```
Comment (renatomefi) [inline README.md]: 3.7 is still possible to use!

File: README.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ All being based on the official images we provide:
 
 - PHP cli - Compiled without php-fpm, a simple php binary image
 - PHP fpm - Specifically designed to share the php-fpm socket towards a fastcgi compliant web sever
-- Nginx - Meant for PHP projects built on the PHP FPM image in this repository, since it looks for a php-fpm socket and doesn't have access to the PHP code
+- Nginx - Meant for PHP projects built on the PHP FPM image in this repository, since it looks for a php-fpm socket and
```
Comment (renatomefi) [inline README.md]: You could link this ADR here: https://github.com/usabilla/php-docker-template/blob/master/docs/adr/0002-nginx-configuration-is-shaped-for-php-needs.md

File: README.md (hunk 1 above)
Comment (cvmiert) [reply README.md]: It's no longer in the list of [supported tags](https://hub.docker.com/_/php?tab=description). Should we reconsider and perhaps drop them from the build matrix?

Comment (renatomefi) [reply README.md]: Hmmm true, we still don't have a policy on cleaning up old tags, I think for now you could just create an issue for it and let's discuss what to do!

Comment (cvmiert) [reply README.md]: > So what was the reason for the dots and new lines in the markdown?
> If it's the md linter I'd suggest to put it in the circleci config thus it won't happen again!

It wasn't any linter, but this just reads a lot more pleasantly regardless of your IDE settings :wink:

**Comment by cvmiert** [comment]
> So what was the reason for the dots and new lines in the markdown?
> If it's the md linter I'd suggest to put it in the circleci config thus it won't happen again!

It wasn't any linter, but this just reads a lot more pleasantly regardless of your IDE settings :wink:

File: README.md (hunk 1 above)
Comment (cvmiert) [reply README.md]: Sounds good!

**Pull request merged**